server/api/reviews.js

```diff
@@ -60,7 +60,7 @@ router.get("/:botId/:reviewId", async (req, res) => {
 });
 
 // Delete ONE review for specified bot
-router.delete("/:botId/reviewId", async (req, res) => {
```

Comment:
Sorry about that. Was just trying to fix the merge conflict but I forgot the colon